cleentfaar/slack-bundle/0.20/config/packages/cleentfaar_slack.yaml

```diff
@@ -0,0 +1,3 @@
+cl_slack:
+    api_token: "%env(SLACK_API_TOKEN)%"
+    test: false
```

Comment:
I'm not sure it should depend on the debug mode. I think this is more likely to change based on the environment.

However, I would add another file enabling test mode for the test environment